src/components/ToggleSwitch.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const ToggleSwitchContainer = styled.div`
+  display: inline-block;
+`
+
+// TODO maybe put background colour in ThemeProvider?
+const ToggleSwitchGraphic = styled.div`
+  width: 35px;
+  height: 30px;
+  background: #4F4A59;
+  z-index: 0;
+  cursor: pointer;
+  position: relative;
+  border-radius: 50px;
+  line-height: 40px;
+  text-align: right;
+  padding: 0 10px;
+  bottom: 10px;
+  color: rgba(0,0,0,.5);
```

Comment:
is this just half opacity black? what is this supposed to do

Reply:
don't think we need this actually